README.md

```diff
@@ -93,6 +93,28 @@ export default function Input() {
 }
 ```
 
+### Using the debounced setter
+
+In addition to getting the debounced value, you can destructure a second element which is a debounced setter function. This allows you to use `useDebounce` without needing a separate state variable for the input value:
```

Comment:
It's not "in addition". It's main feature, explained above. 
If you want to add additional information, let's put simple inside this section https://github.com/xnimorz/use-debounce/blob/master/README.md#simple-values-debouncing
`useDebounce returns a tuple with 2 arguments: debounced value and setter`.